<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>手势识别系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_FirstUse">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>169</y>
      <width>250</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AcadEref</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>第一次使用</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="pushButton_MakeTrain">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>录制训练集</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ChooseTrain">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>导入训练集</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_MakeTest">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>录制测试集</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ChooseTest">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>导入测试集</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_AlreadyHas">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>170</y>
      <width>250</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AcadEref</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>已有数据集</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="pushButton_TrainModel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>模型训练</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_StartUse">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>开始识别</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_Welcome">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AcadEref</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>手势识别系统</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Tips">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>370</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AcadEref</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>操作提示</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
